go/pkg/utils/cache.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+var filePathsFromRoot = make(map[string][]string)
```

Comment:
My understanding is that the cache is a global and shared variable so if there are 2 executions of alizer then that may be incorrect and confusing so I would recommand to have the cache be part of an Alizer execution context that is created for each invocation of Alizer

Reply:
I don't get this. If there are 2 executions of alizer there will be 2 processes so i don't think this will be shared, but even if this happened, the map key is the path to being analyzed so the files list will always be the same. If you are executing 2 analysis at the same time it's unlikely the folder content changes.

Reply:
Alizer can be integrated in 2 ways: Go library or standalone executable. So my remark applies to the first use case